controllers/controller.js

```diff
@@ -10,4 +10,12 @@ res.status(200).send(topics.rows)
 });
 }
 
-module.exports = getAllTopics
\ No newline at end of file
+function getAllEndpoints(req, res, next){
+    selectAllEndpoints().then((response)=>{
+        res.set('Content-Type', 'application/json').status(200).send(response)
```

Comment:
To fit with convention, this `response` should be sent back as an object from the controller on an appropriately-named key. This makes your API easier for other developers to understand ( e.g. { api } or { topics })